src/styles/_page.scss

```diff
@@ -0,0 +1,4 @@
+.page {
```

Comment:
I don't see that you use such a class in html, where did you get it from?

Reply:
I get it now. It's misunderstanding, I used my project in past like notebook and forget remove.